apps/web/src/app/(signing)/sign/[token]/date-field.tsx

```diff
@@ -35,12 +46,20 @@ export const DateField = ({ field, recipient }: DateFieldProps) => {
 
   const isLoading = isSignFieldWithTokenLoading || isRemoveSignedFieldWithTokenLoading || isPending;
 
+  const localDateFormattedValue = convertToLocalSystemFormat(
+    field.customText,
+    dateFormat,
+    timezone,
+  );
+
+  const dateFormatValue = dateFormat === null ? DEFAULT_DOCUMENT_DATE_FORMAT : dateFormat;
```

Comment:
This shouldn't be required anymore, inference should know that it's always assigned I think

Reply:
otherwise we can just `dateFormat ?? DEFAULT_DOCUMENT_DATE_FORMAT`